src/main/resources/application.conf

```diff
@@ -37,26 +37,30 @@ backend {
   backend = "local" // Either "JES" or "local", case insensitive
 
   // If backend is JES, must supply a 'jes' stanza:
-  //      jes
-  //          applicationName
-  //          project (i.e. google project name)
-  //          baseExecutionBucket (GCS path where things will be written to)
+  jes {
+    applicationName = ""
+    project = ""
+    baseExecutionBucket = ""
+  }
 }
 
 // If authenticating with Google (e.g. if backend is 'JES') you must supply a top level stanza 'google':
+google {
+  authScheme = "user"
 
-//      google
-//           authScheme  - Either "service" for service account or "user" for user-based, case insensitive
-                // If "user" you must supply:
-                //      userAuth
-                //           user
-                //           secretsFile
-                //           dataStoreDir (dir to cache oauth credentials)
-                // You will be asked to do the oauth browser dance
+  // If authScheme is "user"
+  userAuth {
+    user = ""
+    secretsFile = ""
+    dataStoreDir = ""
+  }
 
-                // If "service" you must supply:
-                //        p12File
-                //        serviceAccountId
+  // If authScheme is "service"
+  serviceAuth {
+    p12File = ""
+    serviceAccountId = ""
+  }
+}
 
 database {
   config = main.hsqldb
```

Comment:
I'd personally prefer to leave things commented although we'll probably need a better system for documenting "what one could do if they wanted to" in the conf files.

Along those lines I'd be curious to hear why you found it easier to work w/ uncommented, you could certainly convince me off of my comment stance. However the comments are currently intended to show things that you could put in your own personal conf file, you shouldn't ever be directly editing the application.conf.